line_item_manager/operations.py

```diff
@@ -189,6 +189,43 @@ def __init__(self, *args, name: str=None, _type: str='PREDEFINED', **kwargs):
         kwargs['type'] = _type
         super().__init__(*args, **kwargs)
 
+class Geography(AppOperations):
+    service = 'PublisherQueryLanguageService'
```

Comment:
This service is inconsistent with the class name, seems this should be a generic PQL class and supporting methods, so things like 'Geo_Target' are provided on instantiation.  Possible a new 'Geo_Target' class could use the PQL as a parent class.